### PR TITLE
Syncing tab choices using groupId

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Close https://github.com/pingcap/docs-cn/issues/10877

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-delete-data.md [hunk 1]
```diff
@@ -170,8 +170,8 @@ TiDB 使用[统计信息](/statistics.md)来决定索引的选择，因此，在
 
 假设发现在特定时间段内，发生了业务错误，需要删除这期间内的所有 [rating](/develop/dev-guide-bookshop-schema-design.md#ratings-表) 的数据，例如，`2022-04-15 00:00:00` 至 `2022-04-15 00:15:00` 的数据。并且在 15 分钟内，有大于 1 万条数据被写入，此时请使用循环删除的方式进行删除：
 
-<SimpleTab>
-<div label="Java">
+<SimpleTab groupId="language">
```
Comment (ran-huang) [inline develop/dev-guide-delete-data.md:173]: 这里没有 sql 的例子吗？

File: develop/dev-guide-update-data.md [hunk 2]
```diff
@@ -191,7 +191,7 @@ ALTER TABLE `bookshop`.`ratings` ADD COLUMN `ten_point` BOOL NOT NULL DEFAULT FA
 >
 > 此批量更新程序将使用 **DDL** 语句将进行数据表的模式更改。TiDB 的所有 DDL 变更操作全部都是在线进行的，可查看此处，了解此处使用的 [ADD COLUMN](/sql-statements/sql-statement-add-column.md) 语句。
 
-<SimpleTab>
+<SimpleTab groupId="language">
 <div label="Golang">
```
Comment (ran-huang) [inline develop/dev-guide-update-data.md:195]:
```suggestion
<div label="Golang" value="golang">
```

File: develop/dev-guide-delete-data.md (hunk 1 above)
Comment (Oreoxmt) [reply develop/dev-guide-delete-data.md:173]: 没有，这是带事务的批量删除，需要编写程序循环删除

**Review by ran-huang — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**